Dockerfile

```diff
@@ -1,8 +1,31 @@
-FROM centos:7
+FROM alpine:latest
 
-ENV GIN_MODE=release
+RUN echo "@testing http://dl-cdn.alpinelinux.org/alpine/edge/main" >> /etc/apk/repositories \
+  && apk add --no-cache \
+  git \
+  make \
+  musl-dev \
+  go
+#   paxmark bison flex texinfo gawk zip gmp-dev mpfr-dev mpc1-dev zlib-dev libucontext-dev linux-headers isl-dev gcc-gnat-bootstrap
 
-COPY config-example.yaml  /config.yaml
-COPY bin/helm-wrapper /helm-wrapper
+# Configure Go
+ENV GOROOT=/usr/lib/go \
+    GOPATH=/go \
+    PATH=/go/bin:$PATH
+
+RUN  mkdir -p ${GOPATH}/src ${GOPATH}/bin \
+  && git clone https://github.com/opskumu/helm-wrapper $GOPATH/src/github.com/opskumu/helm-wrapper \
+  && cd $GOPATH/src/github.com/opskumu/helm-wrapper \
+  && make build \
+  && make build-linux
```

Comment:
You don't need both lines here. Since both the intermediate and the resulting docker container are based on an alpine linux, both `make` commands should lead to the same result.